lib/ret_web/controllers/page_controller.ex

```diff
@@ -792,4 +792,19 @@ defmodule RetWeb.PageController do
   defp with_script_tags(script), do: "<script>#{script}</script>"
 
   defp module_config(key), do: Application.get_env(:ret, __MODULE__)[key]
+
+  defp maybe_scrub_room_data(hub) do
+    if hub.entry_mode == :deny do
```

Comment:
Probably more idiomatic, not sure if better.. But you could pattern match on the entry_mode instead of using a conditional. As in:
```elixir
defp maybe_scrub_room_data(%Hub{entry_mode: :deny}) do
```